sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -935,7 +941,11 @@ func jsonToMetricsStaticConditionBlock(conditions []TriggerCondition) map[string
 		case "ResolvedCritical":
 			hasCritical = true
 			criticalDict["time_range"] = condition.PositiveTimeRange()
```

Comment:
Yeah, we should be able to remove this block (and the one for warning) altogether
```
			// Issue 297: Do not set parent block's occurrence_type from resolution triggers.
			// Resolution triggers have either the same occurrence_type as their alert counterparts,
			// or, in case of MetricsStaticCondition, are always set to "Always".
			// In either case, the parent's occurrence_type will be set when visiting the alert trigger.
			// criticalDict["occurrence_type"] = condition.OccurrenceType
```

Reply:
done

Reply:
My comment was for `criticalDict["time_range"]` etc, we can skip repeating it for resolved triggers.